docs/pages/management/admin/troubleshooting.mdx

```diff
@@ -161,3 +161,7 @@ cluster, read our [Teleport Diagnostics](../../reference/metrics.mdx) guide.
 For additional sources of Teleport support, please see the
 [Teleport Support and Education Center](https://goteleport.com/support/).
 
+## Common Issues
+
+### `teleport.cluster.local`
```

Comment:
I don't really like this being under "Common Issues" or even in the "Troubleshooting" page at all.

Is there somewhere it would be more discoverable as a piece of "helpful trivia" that relates to all Teleport products ?

Reply:
I think it's an accurate place for it to live. People always see `teleport.cluster.local` and say "huh, what on earth is wrong?"

Reply:
If we wanted to expand on the technicalities a bit we could add it to `architecture/tls-routing` I suppose? I feel like it's important to have it _somewhere_, so searching for `teleport.cluster.local` will result in some explanations.

Reply:
Yeah I mostly care about having it somewhere so that Google/Algoila will return it when you search for `teleport.cluster.local`. I don't have strong opinions on exactly _where_ it goes!